frontend/src/app/lib/addon.js

```diff
@@ -46,8 +46,9 @@ export function installAddon(store, eventCategory, experimentTitle) {
   };
 
   if (useMozAddonManager) {
-    sendToGA('event', gaEvent);
     mozAddonManagerInstall(downloadUrl).then(() => {
+      gaEvent.eventValue = RESTART_REQUIRED ? 1 : 0;
```

Comment:
...yeah. I don't know why this worked when I tested it out, I'm going to backtrack to make sure it's doing what's intended.